assets/js/main.js

```diff
@@ -78,3 +78,23 @@ $(".show-hide").click(function() {
     $(this).removeClass("ms-icon icon-hide").addClass("ms-icon icon-view");
   }
 });
+
+// navigation partials
+
+$('.dpsg-link').on('click', function() {
+  var partial = $(this).attr('data-partial'),
+    elementClicked = this;
+  $.ajax({
```

Comment:
What happens if I click one tab, and then another, and then another, very fast? Perhaps you should ensure only one call happens at each time.